### PR TITLE
DEVX-68 migrate benchmark tests for product sync

### PR DESCRIPTION
**Pull request by lojzatran**
#### Summary

Migrate benchmark tests for product sync to java sdk v2.

JIRA: https://commercetools.atlassian.net/browse/DEVX-68

### CODE REVIEW

File: src/benchmark/java/com/commercetools/sync/benchmark/sdk2/BenchmarkUtils.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          .map(path -> path + "/tmp_git_dir/benchmarks/")
+          .orElse("");
+  private static final String BENCHMARK_RESULTS_FILE_PATH =
+      BENCHMARK_RESULTS_FILE_DIR + BENCHMARK_RESULTS_FILE_NAME;
+  private static final Charset UTF8_CHARSET = StandardCharsets.UTF_8;
+  private static final String EXECUTION_TIME = "executionTime";
+  private static final String TAG_NAME =
+      ofNullable(System.getenv("GITHUB_TAG")).orElse("DEV-SNAPSHOT");
+
+  static final String PRODUCT_SYNC = "productSync";
```
Comment (salander85) [inline src/benchmark/java/com/commercetools/sync/benchmark/sdk2/BenchmarkUtils.java:29]: This is never used and will not in future. Comment or delete it maybe?

**Review by salander85 — APPROVED** [review]

**Comment by salander85** [comment]
Please add tests for the changes, to avoid decreasing coverage.

File: src/benchmark/java/com/commercetools/sync/benchmark/sdk2/BenchmarkUtils.java (hunk 1 above)
Comment (lojzatran) [reply src/benchmark/java/com/commercetools/sync/benchmark/sdk2/BenchmarkUtils.java:29]: Seems like it's being used to publish the results in the benchmark branch. WDYT?

**Comment by lojzatran** [comment]
> Please add tests for the changes, to avoid decreasing coverage.

https://github.com/commercetools/commercetools-sync-java/pull/955/commits/68866865128b1717d8d40b6b08afcac0107e73bb, https://github.com/commercetools/commercetools-sync-java/pull/955/commits/38df7ea84ff867109ed22777bfdd714bb0515290

**Pull request merged**